src/components/LogoutButton.tsx

```diff
@@ -0,0 +1,31 @@
+// 로그아웃 버튼 기능 구현, 세션 삭제
+import { Button as RadixButton } from "@radix-ui/themes"
+import { requestLogout } from "../api"
+
+function LogoutButton(){
+    const handleLogout = async () => {
+        
+        console.log("로그아웃 버튼 눌림");
+
+        sessionStorage.removeItem("access_token");
```

Comment:
```suggestion
        localStorage.removeItem("access_token");
```

엑세스 토큰은 로컬스토리지에 저장합니다.